notebooks/tutorials/1_getting_started_lesson4/notebook.ipynb

```diff
@@ -0,0 +1,406 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Strategy Definition"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now that we have learned how to access and manipulate data in Quantopian, let's construct a data pipeline for our long-short equity strategy. In general, long-short strategies consist of modeling the relative value of assets with respect to each other, and placing bets on the sets of assets that we are confident will increase ([long](https://www.investopedia.com/terms/l/long.asp)) and decrease ([short](https://www.investopedia.com/terms/s/short.asp)) the most in value.  \n",
```

Comment:
`in Quantopian` -> `on Quantopian`
Do we have a general guide for this? I typically say `on`.

Reply:
`long-short strategies` -> `long-short equity strategies`

Reply:
I'd also link to the LS Equity Lecture